docs(android/app): Add help page on using the banner (#10691)
keymanapp/keyman: android/help/basic/using-the-banner.md

---
title: Using the Banner on the Keyboard - Keyman for Android Help
---

## About the Keyboard Banner

Keyman keyboards now always display a banner above the keyboard for one of the following functionalities:

* Display suggestions (See "Using the Suggestion Banner" below)
* Display a Keyman-themed banner so popups and gestures for the top row of keys are visible
* Reserved for future functionality

## Using the Suggestion Banner

If a [dictionary is installed](installing-dictionaries) and enabled for the active Keyman keyboard, the banner will display suggestions that can be selected.

![](../android_images/settings-suggestions-ap.png)

> * The banner can slide when a touch is held down to reveal more suggestions

Reviewer: ```suggestion
* You can slide the banner to see more suggestions by dragging it horizontally with your finger
```

> * Overly-long suggestions are partially hidden, but expand when touched

Reviewer: ```suggestion
* Overly-long suggestions are partially hidden, but expand when you hold your finger on them
```

> * If a suggestion is selected while attempting to scroll the banner, it will not be applied

Reviewer: ```suggestion
* A suggestion will not be applied while you slide the banner
```
